aiotinydb/database.py

```diff
@@ -18,10 +18,13 @@
 
 # pylint: disable=super-init-not-called,arguments-differ
 # pylint: disable=too-many-instance-attributes
+from asyncio import Lock
```

Comment:
I like this appoach, but it's going to be ok only if concurrent access is done from the same process. I think, we should at least consider concurrent access from several processes (my older solution with `AioFileLock`). I'm not sure, if `aiofiles` handles file locking. If we don't provide inter-process locking, then I think we **should** point it out in docs.

Reply:
I wasn't aware that this might be a use case for this library. I implemented a similar wrapper around `fcntl.flock` that gets used if `fcntl` can be imported (so file locking can only be used on unix-like systems). The previous approach is still kept, so a single `AIOTinyDB` instance can also be accessed from multiple coroutines.